content/docs/command-reference/import.md

```diff
@@ -104,15 +104,16 @@ To actually [version the data](/doc/start/data-and-model-versioning), `git add`
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
-- `--no-exec` - create the import `.dvc` file but don't download `url` (assumes
+- `--no-exec` - create the import `.dvc` file without accessing `url` (assumes
   that the data source is valid). This is useful if you need to define the
   project imports quickly, and import the data later (use `dvc update` to finish
   the operation(s)).
 
-- `--no-download` - create the import `.dvc` with data checksums but without
-  downloading the associated data. This is useful if you need track changes in
-  remote data but do not (yet) need to download data to the local workspace.
-  Data can be later downloaded using `dvc pull`.
+- `--no-download` - create the import `.dvc` file including the source data
+  [version](https://git-scm.com/docs/revisions) but without downloading the
```

Comment:
the import .dvc file saves both the repo URL and the revision (Git commit) so maybe we should keep this more general to "source data information" like in https://github.com/iterative/dvc/pull/8341#pullrequestreview-1122585535.